### DISCUSSION TITLE
Regarding the probability or system prompts for legendary NPCs

### DISCUSSION THREAD

**Discussion by KougamiYuriko** [FakeFishGames/Barotrauma#15059] [Ideas and suggestions]
I think the randomness of quests is too unfriendly for single-player mode; I really enjoy collecting these NPCs, but their appearance rates are very low, and there are no key hints. For instance, characters like Artie Dolittle, the Stowaway NPC, and the doctor from the abandoned outpost are legendary NPCs that lack special indicators. If you don’t interact with them when they first appear, they won’t show up again in that campaign. A more noticeable prompt when a legendary NPC is available would be much more helpful.

And if possible, it would be great if special NPCs could be triggered repeatedly. For instance, Ignatius May and Aunt Doris appear multiple times if your reputation is high enough and you don’t talk to recruit them. In contrast, random NPCs only appear once.

By the way, I’ve played for 1,700 hours and have never encountered the Stowaway event NPC. I only found out about this event by looking it up on the wiki. When I found an empty box on the submarine, I realized I’d triggered the event in the previous round but missed the conversation, so that NPC never appeared again in the campaign.

**Comment by AnalogKnight** [comment]
If you don't talk to a special NPC when you first encounter them, they may actually still be encountered again later, and you may not have much luck.
I agree though, even then the probability of them appearing is too small.
